app/src/main/java/com/android/feedme/ui/component/RecipeStepInput.kt

```diff
@@ -0,0 +1,110 @@
+package com.android.feedme.ui.component
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.DeleteForever
+import androidx.compose.material.icons.filled.ExpandLess
+import androidx.compose.material.icons.filled.ExpandMore
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.android.feedme.model.data.Step
+
+@Preview
+@Composable
+fun StepList(
+    modifier: Modifier = Modifier,
+) {
+  val totalSteps = 4
+  LazyColumn(modifier = modifier) {
+    items(totalSteps) { index ->
+      val step = Step(1, "woah", "woa")
+      StepInput(
+          step = step,
+          onStepChanged = { newStep ->
+            // stepViewModel.updateStep(index, newStep)
+          },
+          onDeleteStep = {
+            // stepViewModel.deleteStep(index)
+          })
+    }
+  }
+}
+
+@Composable
+fun StepInput(step: Step, onStepChanged: (Step) -> Unit, onDeleteStep: () -> Unit) {
```

Comment:
Hello, great UI ! Thank you ! Could you just add documentation for the function here please :)

Reply:
Hey, yes, I will add some! 
Thanks